test/appium/views/chat_view.py

```diff
@@ -266,6 +267,7 @@ class ChatView(BaseView):
     def __init__(self, driver):
         super(ChatView, self).__init__(driver)
 
+        self.chat_name = ChatNameText(self.driver)
```

Comment:
'UserNameText' in the chat_view.py has the same accessibility id, maybe we can use it or combine these two elements into one?

Reply:
Well, since these elements exist in two different views I propose to leave both as-is.
The only thing I would change in the line is set `UserNameText` instead of `UserNameText `